quiz.rb

```diff
@@ -2,19 +2,66 @@
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
```

Comment:
Nawal, sorry for taking so long to get your score back to you! I got cut off part-way through this morning and haven't had a chance to finish until just now.